components/slices/Article.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+  <div
+    class="article"
+    :class="{
+      'article--only-text': layout === 'only-text',
+      'row-block': layout === 'text-image' || 'image-text',
+      'row-block--reverse': layout === 'image-text',
+    }"
+  >
+    <div
+      class="article__content"
+      :class="{
+        'article__content--only-text': layout === 'only-text',
+        'row-block__left-content': layout === 'text-image' || 'image-text',
+      }"
+    >
+      <prismic-rich-text :field="title" class="article-content__title" />
+      <prismic-rich-text
+        :field="description"
+        class="article-content__description"
+        :class="{
+          'article-content__description--only-text': layout === 'only-text',
+        }"
+      />
+      <div v-if="linkType !== 'none'">
+        <cta-button
+          v-if="linkType === 'call-to-action'"
+          :link="linkUrl"
+          :name="linkName"
+        />
+        <a
+          v-else-if="linkType === 'link-to'"
+          :href="linkUrl"
+          class="article-content__link-to"
+        >
+          <fa icon="arrow-right" /> {{ linkName }}
+        </a>
+      </div>
+    </div>
+    <prismic-image
+      v-if="layout !== 'only-text'"
+      :field="background"
+      class="row-block__right-content row-block-right-content__background article-illustrations__background"
+    />
+    <prismic-image
+      v-if="layout !== 'only-text'"
+      :field="image"
+      class="row-block__right-content row-block-right-content__image article-illustrations__image"
+    />
+  </div>
+</template>
+
+<script>
+import CtaButton from '../CtaButton'
+
+export default {
+  name: 'Article',
+  components: {
+    CtaButton,
+  },
+  props: {
+    content: {
+      type: Object,
+      default: null,
+    },
+  },
+  computed: {
+    background() {
+      return this.content.primary['article-background']
```

Comment:
**chore:** Modifier le modèle de document et ses chanmps dans Prismic pour suivre les [conventions de nommage proposées sur le wiki](https://1024pix.atlassian.net/wiki/spaces/DEV/pages/1764753451/2020-08-24+R+flexions+sur+des+conventions+de+nommage+dans+Pix+Site).

En plus d'avoir une consistance dans le code, ça permettrait de ne pas avoir à faire de compouted properties "juste à cause" d'un tiret.

Reply:
Convention appliquée
Pour se passer des computed properties, il faudrait mettre directement `this.content.primary.article_background.url` dans le code ? Je trouve que ça alourdi sensiblement la lecture

Reply:
Pour faciliter la lecture, ne pourrait-on pas passer par une seule computed retournant l'ensemble de l'objet 'content' ?

`computed: {
    articleContent() {
    return this.content.primary
    }
}`

Ou passer directement l'ensemble de slice.primary en props dans l'appel de article du component parent SliceZone ?

`<article-slice
       :content="slice.primary"
 >`

et : 

`<prismic-rich-text
        :field="content.description"
 />`